### PR TITLE
README gallery

### PR DESCRIPTION
**Pull request by ColCarroll**
Some minor edits, but mostly adding a gallery to the readme. 

Note that I am linking the images directly to the documentation, instead of internally into the github repo. This is a little risky (if the documentation goes down, the README looks bad), but makes the README portable: the images will still display on pypi, for example.

You can see it at https://github.com/ColCarroll/arviz/tree/pretty-docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,10 +22,115 @@ pip install git+git://github.com/arviz-devs/arviz.git
...
 
 ## Developing
 
-There is a Dockerfile which helps for isolating build problems and local development. Install Docker for your operating system, clone this repo, then run ./scripts/start_container.sh. This should start a local docker container called arviz, as well as a Jupyter notebook server running on port 8888. The notebook should be opened in your browser automatically (you can disable this by passing --no-browser). The container will be running the code from your local copy of arviz, so you can test your changes.
+There is a Dockerfile which helps for isolating build problems and local development.
+1. Install Docker for your operating system
+2. Clone this repo,
+3. Run `./scripts/start_container.sh`
+
+ This should start a local docker container called arviz, as well as a Jupyter notebook server running on port 8888. The notebook should be opened in your browser automatically (you can disable this by passing --no-browser). The container will be running the code from your local copy of arviz, so you can test your changes.
```
Comment (canyon289) [inline README.md]: I would add one line saying you should run tests, and also can lint code with ./lint.sh if desired

**Comment by canyon289** [comment]
You have one committed labels WIP. I think that could have a better commit message!

**Comment by ColCarroll** [comment]
Interesting - I had thought commits were getting squashed, but I see a few silly commit messages. Do you know an easy way to fix that? Maybe we default to doing "squash and merge"?

**Comment by canyon289** [comment]
I believe thats the way to do it. There's these UI buttons. Squash and rebasing like pymc3 sounds reasonable to me

screen shot 2018-09-06 at 6 39 31 am

**Comment by aloctavodia** [comment]
Agree "squash and merge" should be the default

**Comment by canyon289** [comment]
I turned off the merge commit and rebase so there should only be one option now

**Pull request merged**